raiden/tests/integration/network/transport/rif_comms/test_client.py

```diff
@@ -22,23 +21,6 @@ def comms_nodes(amount_of_nodes) -> {int, CommsNode}:
         node.stop()
 
 
-@pytest.mark.parametrize("amount_of_nodes", [1])
-def test_locate_own_peer_id(comms_nodes):
```

Comment:
why it was removed?

Reply:
because the client's `_get_peer_id` method was removed